yucca/training/augmentation/YuccaAugmentationComposer.py

```diff
@@ -204,6 +204,56 @@ def compose_val_transforms(self):
         )
         return val_transforms
 
+    def lm_hparams(self):
+        hparams = {
```

Comment:
I am not sure if this possible, but couldn't we maybe add this as a dict in a dict? In this way when expanding the config dict in wandb, this will not be expanded before also expanding the _augmentations_ subdict. Wandb is sometimes struggling with large dicts.

Reply:
hm yeah I also thought about this when I looked at the hparams, I will do this